lib/chain.js

```diff
@@ -161,7 +161,9 @@ function call(handler, err, req, res, _next) {
         return;
     } else if (!hasError && arity < 4) {
         // request-handling middleware
-        handler(req, res, next);
+        process.nextTick(function nextTick() {
```

Comment:
Have you run the benchmarks with this PR?  Consider this implementation instead:
process.nextTick(handler, req, res, next);
Looking at the implementation of `nextTick` with Julien, it isn't clear which would be faster.

Reply:
yes I run, looks like no effect

Reply:
The goal would be cleaner flamegraphs not faster execution.